cinema/migrations/0003_alter_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-12-16 14:13
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0002_movie_image"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(
+                null=True, upload_to=cinema.models.movie_image_path
```

Comment:
Ensure that the `movie_image_path` function is defined in the `cinema.models` module. This function should implement the image naming format as specified in the task description: `f"{slugify(movie.title)}-{uuid}{ext}"`.